.tekton/kserve-controller-210-pull-request.yaml

```diff
@@ -8,7 +8,7 @@ metadata:
     build.appstudio.redhat.com/target_branch: '{{target_branch}}'
     pipelinesascode.tekton.dev/max-keep-runs: "3"
     pipelinesascode.tekton.dev/on-cel-expression: event == "pull_request" && target_branch
-      == "rhoai-2.10" && (!".tekton/*.*".pathChanged() || ".tekton/kserve-controller-210-pull-request.yaml".pathChanged()) && !"config/overlays/odh/params.env".pathChanged()
+      == "rhoai-2.10" && ( !".tekton/**".pathChanged() || ".tekton/kserve-controller-210-pull-request.yaml".pathChanged() ) && !"config/overlays/odh/params.env".pathChanged()
```

Comment:
@moulalis Can you use the same format I used in the main branch. That's more readable

Reply:
> @moulalis Can you use the same format I used in the main branch. That's more readable

I copied it from main branch file

Reply:
> @moulalis Can you use the same format I used in the main branch. That's more readable

@riprasad I've updated the changes as expected.